env_setup/handle_existing_vm.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Not cleanup old snapshot by default"
```

Comment:
I wonder whether we can revert the base snapshot firstly if it exists for existing VM. Then no matter old snapshots are cleaned or not, below `vm_get_vm_info.yml` will always be the VM info at base snapshot or the VM info without base snapshot. And we don't need to get it again in `test_setup.yml`, and we can also move `esxi_get_guest_config_options.yml` from `test_setup.yml` to this file.

Reply:
This we can take a look at all the related tasks and consider later. Thanks.